### PR TITLE
Refs #24033 - Refactor subscriptions search-query

### PR DESCRIPTION
**Pull request by sharvit**
Move the search-query state from the SubscriptionsPage to redux

### CODE REVIEW

File: webpack/scenes/Subscriptions/SubscriptionsPage.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ class SubscriptionsPage extends Component {
       subscriptionDeleteModalOpen: false,
       disableDeleteButton: true,
       showTaskModal: false,
-      searchQuery: '',
+      // searchQuery: '',
```
Comment (waldenraines) [inline webpack/scenes/Subscriptions/SubscriptionsPage.js]: Can we just remove this line?

**Comment by sharvit** [comment]
Thanks @waldenraines, done.

**Comment by waldenraines** [comment]
[test katello]

**Comment by waldenraines** [comment]
@sharvit sorry, I created some conflicts for you here by merging #7833.

**Comment by sharvit** [comment]
np @waldenraines, rebased.

File: package.json [hunk 2]
```diff
@@ -77,8 +77,13 @@
...
     "collectCoverage": true,
-    "collectCoverageFrom": ["webpack/**/*.js", "!webpack/**/bundle*"],
-    "coverageReporters": ["lcov"],
+    "collectCoverageFrom": [
+      "webpack/**/*.js",
+      "!webpack/**/bundle*"
+    ],
+    "coverageReporters": [
+      "lcov"
+    ],
```
Comment (sharvit) [inline package.json]: I'm not sure why but when I run `npm test` or `npm i` it changes the style here :/

**Pull request merged**